tests/testthat/_snaps/batch-chat.md

```diff
@@ -0,0 +1,37 @@
+# errors if chat/provider/prompts don't match previous run
+
+    Code
+      batch_chat(chat, prompts, path)
+    Condition
+      Error in `batch_chat()`:
+      ! provider, prompts, and user_turns don't match stored values.
```

Comment:
Should the error message include the path value to make it easier to track down the problematic file?